src/components/ui/links/index.jsx

```diff
@@ -1,7 +1,9 @@
 import React from "react";
+
 import { Link } from "react-router-dom";
+import { attachClasses } from "../../../utils/index";
```

Comment:
1. This import statement should be after the classes import
2.  No need to add index in ../../../utils/index - only ../../../utils will work fine